src/octokit_utils/index.ts

```diff
@@ -1,127 +1,181 @@
 }
 
-async function approveChange(context: Context<Webhooks.WebhookPayloadPullRequest>): Promise<void> {
-  const req = initPullRelatedRequest(context)
-  req.event = 'APPROVE'
-  context.log.info(`Reviewing PR with request ${JSON.stringify(req)}`)
+async function approveChange(
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+): Promise<void> {
+  const req = initPullRelatedRequest(context);
+  req.event = "APPROVE";
+  context.log.info(`Reviewing PR with request ${JSON.stringify(req)}`);
   try {
-    const res = await context.github.pulls.createReview(req)
+    const res = await context.github.pulls.createReview(req);
     if (res.status === 200) {
-      context.log.info('Approve changes succeeded.')
+      context.log.info("Approve changes succeeded.");
     } else {
-      context.log.error(`Approve change rejected with: ${JSON.stringify(res.data)}`)
+      context.log.error(
+        `Approve change rejected with: ${JSON.stringify(res.data)}`,
+      );
     }
   } catch (err) {
-    context.log.error(`Approve change failed with: ${JSON.stringify(err)}`)
+    context.log.error(`Approve change failed with: ${JSON.stringify(err)}`);
   }
 }
 
-async function createPassingStatus(context: Context<Webhooks.WebhookPayloadPullRequest>, startTime: string, endTime: string): Promise<void> {
-  const statusOptions: (Octokit.RequestOptions & Octokit.ChecksCreateParams) = context.repo({
+async function createPassingStatus(
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+  startTime: string,
+  endTime: string,
+): Promise<void> {
+  const statusOptions: Octokit.RequestOptions &
+    Octokit.ChecksCreateParams = context.repo({
     name: APP_CHECK_NAME,
     head_sha: context.payload.pull_request.head.sha,
-    status: 'completed',
+    status: "completed",
     started_at: startTime,
     completed_at: endTime,
-    conclusion: 'success',
+    conclusion: "success",
     output: {
-      title: 'test',
-      summary: 'test',
-      text: 'test',
+      title: "test",
+      summary: "test",
+      text: "test",
     },
     request: {
       retries: 3,
       retryAfter: 3,
     },
-  })
+  });
   const response = await context.github.checks.create(statusOptions);
-  context.log.info(`Create passing status finished with status ${response.status}`);
+  context.log.info(
+    `Create passing status finished with status ${response.status}`,
+  );
   if (response.status !== 201) {
-    context.log.error(`Create passing status failed with status ${response.status} and error: ${JSON.stringify(response.data)}`)
+    context.log.error(
+      `Create passing status failed with status ${
+        response.status
+      } and error: ${JSON.stringify(response.data)}`,
+    );
   }
 }
 
-async function getChangedFiles(context: Context<Webhooks.WebhookPayloadPullRequest>) {
-  const changedFilesResponse = await context.github.pulls.listFiles(initPullRelatedRequest(context))
-  const changedFiles: string[] = []
+async function getChangedFiles(
```

Comment:
Codacy found an issue: [expected call-signature: 'getChangedFiles' to have a typedef](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)